testing/scenario_app/bin/run_android_tests.dart

```diff
@@ -311,27 +327,26 @@ Future<void> _run({
       final (Future<int> logcatExitCode, Stream<String> logcatOutput) = getProcessStreams(logcatProcess);
 
       logcatProcessExitCode = logcatExitCode;
+      String? filterToProcessId;
```

Comment:
I found this name really confusing.  The gist is that it is the process id we are filtering for right `filterProcessId` seems more clear to me.

Reply:
Yeah I agree, cleaned it up a bit and added comments.